ql/src/test/queries/clientpositive/udf_mask_hash_config.q

```diff
@@ -0,0 +1,18 @@
+set hive.query.results.cache.enabled=false;
+
+-- create a test table
+create table testhivemask(name string);
+
+-- insert some values
+insert into testhivemask values('name1'),('name2');
+
+-- run the query with default values
+select mask_hash(name) from testhivemask;
+
+-- explicitily configure sha512 and check for values
+set hive.masking.algo=sha512;
+select mask_hash(name) from testhivemask;
+
+-- explicitily configure sha256 and check for values
+set hive.masking.algo=sha256;
+select mask_hash(name) from testhivemask;
```

Comment:
Can you set hive.fetch.task.conversion=none; 
and then set hive.masking.algo=sha512;
and then do a select mask_hash(name) from testhivemask;
So that this makes sure tez container receives hive conf on the run time?

Reply:
Added here, these lines are below checking for sha256, like if both 512 and 256 can work 
https://github.com/ayushtkn/hive/blob/28d86d4c758852d4a89c0eca4eb72873351074a0/ql/src/test/queries/clientpositive/udf_mask_hash_config.q#L9-L14

Reply:
Got a test failure as well, if there is a query like select mask_hash('01-28-2021'); the property wasn't being propagated
Made changes to make it work, tried locally as well as on a cluster, the change worked for me, will wait for the jenkins result